extmod/uzlib/defl_static.h

```diff
@@ -0,0 +1,13 @@
+struct Outbuf {
```

Comment:
Does this file have any copyright/license?  I see it originally came from PuTTY.

Reply:
Well, it definitely contains some prototypes/declarations for things in defl_static.c. But: it has changes/additions beyond what was originally in putty, e.g. changes turned from static to non-static. And overall, I don't think that just a few declarations can be subject to the copyright of the original code. Beyond that, putting the exact blurb from defl_static.c might have a connotation that API as defined in  defl_static.h (and defining an API is the purpose of that header) was done by Simon Tatham, but that borders on impersonation, because original those were internal static functions, and he might not agree, and would never have done something like that.

At the same time, I'm reluctant to put uzlib's blurb by the reason that it indeed comes from the result of processing of putty's code. So, my choice would be leave as is.

Reply:
Ok, but if someone in the future asks "what is the copyright and licensing of this file", what is the answer?  Do they assume that, because it has nothing, it is covered by the LICENSE file in the root of this repository, or that it's in the public domain, or that it's too small to have meaningful copyright?  Whatever the case, it would be good to explicitly state it (to avoid, eg, someone not being able to use the code because copyright/licensing is unclear).

Reply:
> Do they assume that, because it has nothing, it is covered by the LICENSE file in the root of this repository

Well, it's definitely covered by the LICENSE file in the root of its original repository - which I also added based on your feedback, but was too shy to include in this patch because: a) not trying to add more bloat to uPy repo; b) not trying to to make uPy a license zoo (always trying the opposite).

> it's too small to have meaningful copyright?

That's my approach, yeah. But many "enterprisey" project indeed have a policy that even the most trivial files have a licensing header. In that regard, yes, I'd add uzlib's licensing header to it.

I'm going to release 2.9.2 with that.